styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 330px;
+  margin: auto;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+}
+
+a {
+  text-decoration: none;
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  background: #fff;
+  border-radius: 16px;
+  max-width: 327px;
```

Comment:
- [ ] I suggest using 100% here too to make sure it fits all the possible screens :recycle: